react/index.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import './index.scss';
+import App from './App';
+
+ReactDOM.render(<App />, document.getElementById('root'));
```

Comment:
Or this one

Reply:
We need this for the js bundle entrypoint